npm/plugin/main.go

```diff
@@ -67,5 +68,7 @@ func main() {
 		panic(err.Error)
 	}
 
+	metrics.StartHTTP(false, 0)
```

Comment:
Shouldnt this be the firt call (with a go routine) before we start processing the rules already?

Reply:
The metrics are recorded in the same way whether or not they're available on an HTTP server. Having it last only means scrapers can't see the metrics until NPM is finished booting up.